packages/components/src/internal/components/samples/SamplesEditButton.tsx

```diff
@@ -76,35 +77,15 @@ export const SamplesManageButton: FC<OwnProps & SampleGridButtonProps & Requires
                 PermissionTypes.EditStorageData,
             ]}
         >
-            <ManageDropdownButton id="samples-manage-btn">
-                {props.children}
-                {shouldShowButtons(SamplesManageButtonSections.IMPORT, hideButtons) && showImportDataButton && (
-                    <RequiresPermission perms={PermissionTypes.Insert}>
-                        <MenuItem href={importSampleHref}>Import Samples</MenuItem>
-                    </RequiresPermission>
-                )}
-                {shouldShowButtons(SamplesManageButtonSections.DELETE, hideButtons) && (
-                    <RequiresPermission perms={PermissionTypes.Delete}>
-                        <SampleDeleteMenuItem
-                            queryModel={model}
-                            afterSampleDelete={afterSampleDelete}
-                            metricFeatureArea={metricFeatureArea}
-                        />
-                    </RequiresPermission>
-                )}
-                {shouldShowButtons(SamplesManageButtonSections.PICKLIST, hideButtons) && (
-                    <RequiresPermission perms={PermissionTypes.ManagePicklists}>
-                        <AddToPicklistMenuItem queryModel={model} user={user} metricFeatureArea={metricFeatureArea} />
-                    </RequiresPermission>
-                )}
-                {shouldShowButtons(SamplesManageButtonSections.EDIT, hideButtons) && (
+            <ManageDropdownButton id="samples-manage-btn" title="Edit">
+                {showEdit && (
                     <RequiresPermission
                         perms={[PermissionTypes.Update, PermissionTypes.EditStorageData]}
                         permissionCheck="any"
                     >
                         <SelectionMenuItem
                             id="update-samples-menu-item"
-                            text="Edit Selected Samples in Grid"
+                            text="Edit in Grid"
```

Comment:
Should there be a check for `user.canUpdate` with this item?

Reply:
I believe it is not there because there are multiple tabs within the "Edit in Grid" and a user might have perm to edit the storage data without being able to edit the sample data. So I believe that component will do the proper checks.